src/components/containers/profile.container.js

```diff
@@ -1,16 +1,17 @@
 import { connect } from 'react-redux';
 import Profilepage from '../presentations/Profilepage/ProfilePage';
 import profileAction from '../../actions/profile.action';
-import fetchArticles from '../../actions/articles-update.actions';
 import followeeAction from '../../actions/followee.actions';
 import followingAction from '../../actions/following.actions';
 import fetchBookmarks from '../../actions/bookmarked.action';
 import RequestAction from '../../actions/reviewers-request.action';
+import myArticlesAction from '../../actions/articles-update.actions';
```

Comment:
A better name for this variable would be nice.

Reply:
Like `userArticle` ?

Reply:
Yes, `userArticle` is a more suitable variable name.